index.html

```diff
@@ -0,0 +1,499 @@
+          <div class="person-para color-black"></div>
+          <div class="person-para color-black"></div>
+          <div class="person-para-last color-black"></div>
+        </div>
+        <div class="jump-button">
+          <button type="submit" class="large book-button">jump button</button>
+        </div>
+      </div>
+      <div class="design-book rotate25 clearrotate color-black">book image</div>
+    </div>
+
+    <!-- section for the email-newsletter -->
+    <div class="email-grid-container sec-color">
+      <div class="email-grid eitems">
+        <div class="smashing-book person-title title">smashing book</div>
+        <div class="design-system top-article color-black">design systems</div>
+        <div class="meet-design">
+          <div class="person-para color-black"></div>
+          <div class="person-para color-black"></div>
+          <div class="person-para-last color-black"></div>
+        </div>
+        <div class="jump-button">
+          <button type="submit" class="large book-button">jump button</button>
+        </div>
+      </div>
+      <div class="email-grid eitems">
+        <div class="smashing-book person-title title">smashing book</div>
+        <div class="design-system top-article color-black">design systems</div>
+        <div class="meet-design">
+          <div class="person-para color-black"></div>
+          <div class="person-para color-black"></div>
+          <div class="person-para-last color-black"></div>
+        </div>
+        <div class="jump-button">
+          <button type="submit" class="large book-button">jump button</button>
+        </div>
+      </div>
+    </div>
+    <div class="newsletter-grid-container sec-color">
+      <div class="email-grid yourlost">
+        <div class="smashing-book person-title title">smashing book</div>
+        <div class="design-system top-article color-black">design systems</div>
+        <div class="meet-design">
+          <div class="person-para color-black"></div>
+          <div class="person-para color-black"></div>
+          <div class="person-para-last color-black"></div>
+        </div>
+        <div class="jump-button">
+          <button type="submit" class="large book-button">jump button</button>
+        </div>
+      </div>
+      <div class="email-grid newsletter flex-col-aic">
+        <div class="news-griditem1">
+          <div class="news-picture fifth-color"></div>
+        </div>
+        <div class="news-griditem">
+          <article>
+            <div class="content"></div>
+          </article>
+        </div>
+        <div class="news-griditem">
+          <article>
+            <div class="content"></div>
+          </article>
+        </div>
+        <div class="news-griditem">
+          <article>
+            <div class="content"></div>
+          </article>
+        </div>
+      </div>
+    </div>
+
+    <!-- footer section starts here -->
+    <div class="footer flex-col-aic fourth-color">
```

Comment:
- [OPTIONAL] You can consider the use of the [`<footer>`](https://www.w3schools.com/TagS/tag_footer.asp) tag for the footer